const.go

```diff
@@ -62,6 +62,26 @@ const (
 	HyperVGeneration2 = "V2"
 )
 
+// Reference - https://docs.microsoft.com/en-us/azure/virtual-machines/vm-naming-conventions
+const (
+	// FamilyKey represents the key for the 'family' part in the SKU size map.
+	FamilyKey = "family"
+	// SubFamilyKey represents the key for the 'subFamily' part in the SKU size map.
+	SubFamilyKey = "subFamily"
+	// VCPUsKey represents the key for the 'vCPUs' part in the SKU size map.
+	VCPUsKey = "vCPUs"
+	// ConstrainedVCPUsKey represents the key for the 'constrainedVCPUs' part in the SKU size map.
+	ConstrainedVCPUsKey = "constrainedVCPUs"
+	// AdditiveFeaturesKey represents the key for the 'additiveFeatures' part in the SKU size map.
+	AdditiveFeaturesKey = "additiveFeatures"
+	// AcceleratorTypeKey represents the key for the 'acceleratorType' part in the SKU size map.
+	AcceleratorTypeKey = "acceleratorType"
+	// VersionKey represents the key for the 'version' part in the SKU size map.
+	VersionKey = "version"
+	// PromoKey represents the key for the 'promo' part in the SKU size map.
+	PromoKey = "promo"
+)
+
```

Comment:
Not sure we need these (see my other comment on using map vs. struct), but if we do - they probably belong in the file where they are used.